config/crd/bases/external-secrets.io_externalsecrets.yaml

```diff
@@ -1,10 +1,10 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
The `apiextensions.k8s.io/v1` API is available since v1.16 where v1beta1 was deprecated. It will be removed in 1.22.
1.16 is not supported anymore and since 2020-09-02 EOL ([source](https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md#non-active-branch-history)).

see: https://github.com/kubernetes/kubernetes/issues/82022

I think we should be fine targeting v1 only. What do you think?

Reply:
I think that's alright!

Was this manually changed on the CRD or generated by the new `controller-gen` version?

Reply:
this was generated by the new version of `controller-gen`.